### DISCUSSION TITLE
install fails

### DISCUSSION THREAD

**Discussion by APellie** [fabianishere/udm-iptv#226] [Q&A]
Hi Fabian,

Copied the install string...but receiving multiple errors. Any idea what goes wrong?
Running UniFi OS UDM Pro 2.4.27
See details below:

root@PelliesWorld:/# sh -c "$(curl https://raw.githubusercontent.com/fabianishere/udm-iptv/master/install.sh -sSf)"
Downloading packages...
Installing packages...
Hit:1 http://security.debian.org stretch/updates InRelease
Ign:2 http://deb.debian.org/debian stretch InRelease
Hit:3 http://deb.debian.org/debian stretch-updates InRelease
Hit:4 http://deb.debian.org/debian stretch-backports InRelease
Hit:5 http://deb.debian.org/debian stretch Release
Ign:6 https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_9  InRelease
Ign:7 https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_9  Release
Ign:8 https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_9  Packages
Ign:8 https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_9  Packages
Ign:8 https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_9  Packages
Ign:8 https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_9  Packages
Ign:8 https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_9  Packages
Err:8 https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_9  Packages
  404  Not Found
Hit:9 https://apt-release-candidate.artifacts.ui.com stretch InRelease
Hit:11 https://apt.artifacts.ui.com stretch InRelease
Hit:12 https://apt-beta.artifacts.ui.com stretch InRelease
Reading package lists...
W: The repository 'https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_9  Release' does not have a Release file.
E: Failed to fetch https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_9/Packages  404  Not Found
E: Some index files failed to download. They have been ignored, or old ones used instead.

**Comment by fabianishere** [comment]
> W: The repository 'https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_9 Release' does not have a Release file.
E: Failed to fetch https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_9/Packages 404 Not Found

This is an issue with one of the apt repositories you have configured. I would look into (temporarily) disabling it.

**Reply by APellie** [reply]
disabled it, used it in the past for something, my mistake. 
install now works...testing IPTV now :-)

**Reply by APellie** [reply]
yes, works again. Thanks @fabianishere and @reinierjh for thinking along.

**Comment by reinierjh** [comment]
I find it strange that there is an OpenSuse repository on your UDM. On my UDM these are the "Official" repositories. And ofcourse the only ones that are supported by Ubiquiti (and i have EA access and therfore the Beta parts in the apt.lists):

root@UDM-SE:~# cd /etc/apt/
root@UDM-SE:/etc/apt# more sources.list
deb http://deb.debian.org/debian/ bullseye main contrib non-free
deb http://deb.debian.org/debian/ bullseye-updates main contrib non-free
deb http://deb.debian.org/debian/ bullseye-backports main
deb http://security.debian.org/debian-security bullseye-security main contrib non-free
root@UDM-SE:/etc/apt# cd sources.list.d/
root@UDM-SE:/etc/apt/sources.list.d# more ubiquiti.list
deb https://apt-beta.artifacts.ui.com/ bullseye beta
deb https://apt-release-candidate.artifacts.ui.com/ bullseye release-candidate
deb https://apt.artifacts.ui.com/ bullseye release
